pico_w/wifi/http_client/mbedtls_config.h

```diff
@@ -0,0 +1,66 @@
+/* Workaround for some mbedtls source files using INT_MAX without including limits.h */
+#include <limits.h>
+
+#define MBEDTLS_NO_PLATFORM_ENTROPY
```

Comment:
This may be a stupid question, but is it worth using another `..._examples_common.h` file here, similar to what you did in #460 ?

Reply:
yes, probably. But not as part of this change